DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Dashboard/RankingController.cs

```diff
@@ -13,19 +17,31 @@
     public class RankingController : Controller
     {
         private readonly ICentresService centresService;
+        private readonly IRegionDataService regionDataService;
 
-        public RankingController(ICentresService centresService)
+        public RankingController(
+            ICentresService centresService,
+            IRegionDataService regionDataService
+        )
         {
             this.centresService = centresService;
+            this.regionDataService = regionDataService;
         }
 
-        public IActionResult Index()
+        public IActionResult Index(int? regionId = null, int? periodId = null)
         {
+            Enumeration.TryGetFromIdOrName<Period>(periodId.ToString() ?? string.Empty, out var period);
+            period ??= Period.Fortnight;

```

Comment:
Because we've already set up the model binder for this I'd expect us to be able to specify Index(int? regionId = null, Period? period = null), like it's done on Support pages.